app/src/main/java/com/imaginaryrhombus/proctimer/TimerApplication.kt

```diff
@@ -0,0 +1,29 @@
+package com.imaginaryrhombus.proctimer
+
+import android.app.Application
+import com.imaginaryrhombus.proctimer.application.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/fizxive/ProcTimer/blob/85662f57ce505e613f53a97ddbeebce37ccafc7f/app/src/main/java/com/imaginaryrhombus/proctimer/TimerApplication.kt#L4"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>